src/actions/__tests__/cursorUp.ts

```diff
@@ -171,6 +178,28 @@ describe('normal view', () => {
     const cursor = state.cursor
     expect(pathToContext(state, prevThought(state, cursor!)!)).toEqual(['a'])
   })
+
+  it('move cursor from empty thought to previous thought in context sorted in descending order', () => {
```

Comment:
Is there a reason you did a full shortcut test instead of a reducer test? Generally reducer tests are going to be faster and more minimal because they do not require `act`.

Reply:
For accuracy, I did full shortcut test.